control-plane/connect-inject/common/openshift.go

```diff
@@ -0,0 +1,129 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+// Function copied from:
+// https://github.com/openshift/apiserver-library-go/blob/release-4.17/pkg/securitycontextconstraints/sccmatching/matcher.go
+// Apache 2.0 license: https://github.com/openshift/apiserver-library-go/blob/release-4.17/LICENSE
+
+// A namespace in OpenShift has the following annotations:
+// Annotations:  openshift.io/sa.scc.mcs: s0:c27,c4
+//               openshift.io/sa.scc.uid-range: 1000710000/10000
+//               openshift.io/sa.scc.supplemental-groups: 1000710000/10000
+//
+// Note: Even though the annotation is named 'range', it is not a range but the ID you should use. All pods in a
+// namespace should use the same UID/GID. (1000710000/1000710000 above)
+
+package common
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/hashicorp/consul-k8s/control-plane/connect-inject/constants"
+	corev1 "k8s.io/api/core/v1"
+)
+
+// Get the user id from the OpenShift annotation 'openshift.io/sa.scc.uid-range'.
+func GetOpenShiftUID(ns *corev1.Namespace) (int64, error) {
```

Comment:
nit: Since this is a public method (and you have a function comment), should put the name first in the function comment, same for below